pkg/test/runner/config.go

```diff
@@ -23,13 +23,39 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// EvalTestCaseConfig contains the config only for imperative
+// function run
+type EvalTestCaseConfig struct {
+	// ExecPath is a path to the executable file that will be run as function
+	// Mutually exclusive with Image
+	ExecPath string `json:"execPath,omitempty" yaml:"execPath,omitempty"`
+	// Image is the image name for the function
+	Image string `json:"image,omitempty" yaml:"image,omitempty"`
+	// Args are the arguments that will be passed into function.
+	// Args will be passed as 'key=value' format after the '--' in command.
+	Args map[string]string `json:"args,omitempty" yaml:"args,omitempty"`
+	// Network indicates is network accessible from the function container. Default: false
+	Network bool `json:"network,omitempty" yaml:"network,omitempty"`
+	// FnConfig is the path to the function config file.
+	FnConfig string `json:"fnConfig,omitempty" yaml:"fnConfig,omitempty"`
+}
+
 // TestCaseConfig contains the config information for the test case
 type TestCaseConfig struct {
-	ExitCode int  `json:"exitCode,omitempty" yaml:"exitCode,omitempty"`
-	Network  bool `json:"network,omitempty" yaml:"network,omitempty"`
-	RunCount int  `json:"runCount,omitempty" yaml:"runCount,omitempty"`
-	Skip     bool `json:"skip,omitempty" yaml:"skip,omitempty"`
-	Debug    bool `json:"debug,omitempty" yaml:"debug,omitempty"`
+	// ExitCode is the expected exit code from the kpt commands. Default: 0
+	ExitCode int `json:"exitCode,omitempty" yaml:"exitCode,omitempty"`
+	// RunCount is how many times the test will run. Each iteration will be
+	// immediately run after previous one. Default: 1
+	RunCount int `json:"runCount,omitempty" yaml:"runCount,omitempty"`
+	// Skip means should this test case be skipped. Default: false
+	Skip bool `json:"skip,omitempty" yaml:"skip,omitempty"`
+	// Debug means will the debug behavior be enabled. Default: false
```

Comment:
what's debug behavior?

Reply:
Currently it's keeping the temp directory used to run the test case. But maybe more things in future.

Reply:
Added the description.